lib/strategy/EnrichClientFromRemoteConfigStrategy.js

```diff
@@ -44,35 +47,53 @@ EnrichClientFromRemoteConfigStrategy.prototype._resolveApplicationByName = funct
   });
 };
 
-EnrichClientFromRemoteConfigStrategy.prototype.process = function (config, callback) {
-  var application = config.application;
+// If there are only two applications and one of them is
```

Comment:
I didn't know that :) But why isn't this used as default when there are no other applications?